l10n/nl/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,1522 @@
+# Translations template for seedsigner.
+# Copyright (C) 2025 ORGANIZATION
+# This file is distributed under the same license as the seedsigner project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2025.
+# 
+# Translators:
+# Dirkson643, 2025
+# Rachid, 2025
+# 
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: seedsigner 0.8.5-rc1\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2025-01-28 13:28-0600\n"
+"PO-Revision-Date: 2022-04-29 13:50+0000\n"
```

Comment:
Just a note that this field is never updated by transifex, but you can see how many hours ago translations were worked on in the dialog for downloading the "for use" messages.po file.   Perhaps it's worth editing, but it's just a comment.

Also, up above, I see no actual email addresses, but depending on profile settings for translators, private emails can get leaked in this document (both above and in the LastTranslator comment.  Perhaps also worth scanning to make sure no privacy leaks unless the translator actually intends to do so.

Reply:
Thanks for checking this!
- I just updated this revision date
- Yeah, we don't want to share our email addresses. We can be contacted via Github or transifex or telegram ;)